.darglint

```diff
@@ -0,0 +1,4 @@
+[darglint]
```

Comment:
Please add a comment on first line mentioning that flake8 is using this via wemake. So a random visitor will be able to identify how is this used.

Reply:
Actually, I've checked that the issue is visible if you just invoke `darglint` directly. I wonder why Brad hasn't faced this locally.

Reply:
I was only running darglint with select files and directories, so I probably just didn't hit it

Reply:
Filed the upstream issue: https://github.com/terrencepreilly/darglint/issues/186.

Reply:
The second issue: https://github.com/wemake-services/wemake-python-styleguide/issues/2287.